browser/admin/src/AdminSocketAnalytics.js

```diff
@@ -23,6 +23,7 @@ var AdminSocketAnalytics = AdminSocketBase.extend({
 	_cpuStatsData: [],
 	_sentStatsData: [],
 	_recvStatsData: [],
+    _connStatsData: [],
```

Comment:
the indents look a bit wonky wrt the surrounding code

Reply:
done (replace to hard TS)